Bottom: 4281bcc6a82f5f962610fdb19afca40a721cbb51
Top:    5e7c36180b4aa98f4d1708cb64dda41635834788
Author: Ian Kent <raven@themaw.net>
Date:   2013-02-27 13:25:39 +0800

Refresh of autofs-5.0.7-fix-requires-in-spec-file.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 1199759..8fa31fc 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -32,6 +32,7 @@
 - fix file descriptor leak when reloading the daemon.
 - depricate nosymlink pseudo option.
 - add symlink pseudo option.
+- fix requires in spec file.
 
 25/07/2012 autofs-5.0.7
 =======================
diff --git a/autofs.spec b/autofs.spec
index b8a3b7a..703f7a9 100644
--- a/autofs.spec
+++ b/autofs.spec
@@ -25,8 +25,8 @@ Buildroot: %{_tmppath}/%{name}-tmp
 %if %{with_systemd}
 BuildRequires: systemd-units
 %endif
-BuildPrereq: autoconf, hesiod-devel, openldap-devel, bison, flex, cyrus-sasl-devel
-Prereq: chkconfig
+BuildRequires: autoconf, hesiod-devel, openldap-devel, bison, flex, cyrus-sasl-devel
+Requires: chkconfig
 Requires: /bin/bash mktemp sed textutils sh-utils grep /bin/ps
 %if %{with_systemd}
 Requires(post): systemd-sysv
